ruby-bubble/ruby-bubble-sort/bubblesort.rb

```diff
@@ -0,0 +1,20 @@
+def bubble_sort(arr)
```

Comment:
r = ["hi", "hello","hey"]
```print bubble_sort_by(r) {|left, right| right.length - left.length} = ["hi","hey","hello"]```
```print bubble_sort_by(r) {|left, right| right.length - left.length} != ["hello","hey","hi"]```

Reply:
Hi Forinson,

Reply:
Hey there,

am sorry but in the project specifications, they have left.length - right.length and not right.length - left.length

Reply:
Could you please elaborate? because I just checked with the project specs and it works fine.
I used the following to test : 
```
print bubble_sort_by(%w[hi hello hey]) { |left, right|
  left.length - right.length
}
```

Reply:
even @whiz25 's code : 

```
words = %w[hi hey hello]

print bubble_sort_by(words) { |left, right| left.length - right.length }
```
works fine now. Could you please check again